### ISSUE TITLE
Problems with the getting started documentation

### ISSUE THREAD

**Issue by kiuka** [api-platform/api-platform#1793]
**API Platform version(s) affected**: 2.6.1

**Description**  
I'm learning Api platform and started with the "Get started" tutorial.

Everything seems fine until I get to the frontend / admin part (https://api-platform.com/docs/distribution/#the-admin), the admin is available, although I cannot create or edit any object because the button never gets active due to false validation errors:

![image](https://user-images.githubusercontent.com/11283191/106892839-f1cd2300-66ec-11eb-9f38-3067d6ecfb0f.png)

I tried to ignore this and continue the tutorial, but the very next step also fails (https://api-platform.com/docs/distribution/#a-nextjs-web-app):

```
docker-compose exec pwa     generate-api-platform-client --resource books
Usage: generate-api-platform-client entrypoint outputDirectory

Generate apps built with Next, Nuxt, Quasar, React, React Native, Vue or Vuetify for any API documented using Hydra or OpenAPI

Options:
  -V, --version                                 output the version number
  -r, --resource [resourceName]                 Generate CRUD for the given resource
  -p, --hydra-prefix [hydraPrefix]              The hydra prefix used by the API (default: "hydra:")
  --username [username]                         Username for basic auth (Hydra only)
  --password [password]                         Password for basic auth (Hydra only)
  --bearer [bearer]                             Token for bearer auth (Hydra only)
  -g, --generator [generator]                   The generator to use, one of "next", "nuxt", "quasar", "react", "react-native", "typescript", "vue", "vuetify" (default: "next")
  -t, --template-directory [templateDirectory]  The templates directory base to use. Final directory will be ${templateDirectory}/${generator} (default:
                                                "/usr/local/share/.config/yarn/global/node_modules/@api-platform/client-generator/lib/../templates/")
  -f, --format [hydra|openapi3|openapi2]        "hydra", "openapi3" or "openapi2" (default: "hydra")
  -s, --server-path [serverPath]                Path to express server file to allow route dynamic addition (Next.js generator only)
  -h, --help                                    display help for command
```

Since it needs to access the openapi, I managed to make it work like this which generated the pages for me:

```
# using the caddy container and use the current directory as the output
docker-compose exec pwa \
    generate-api-platform-client http://caddy .
```

Somehow the whole docker setup is really slow, I'm using linux, if I start to refresh the https://localhost/books (which was generated above - I could use the /admin as well, same result) it stucks most of the time, just loading infinite, waiting for javascript files. Then I refresh again and suddenly it loads fast then refresh again and doesn't load again..

Sometimes I stop the containers and then start them again which gives me a temporary solution (1-2 page loads) but then it start to be slow or unavailable again. I'm not changing the code or anything, so I guess its not about regenerating the code.

Any idea what am I missing or what did I do wrong? I really like the idea of getting all the components out of the box and I guess it should work like a charm since many developer using it:)

**How to reproduce**  
Just follow the tutorial

**Possible Solution**  
Suggested above.

**Additional Context**

**Comment by ksom** [comment]
@kiuka Same problem here with the default Greeting object. Try to create new one, but even if I enter a name that always trigger a required error on the name field.

**Comment by alanpoulain** [comment]
It will be fixed with the 2.6.2 release.

**Comment by ksom** [comment]
@alanpoulain Is there a release date scheduled ?

**Comment by alanpoulain** [comment]
Very soon!

**Comment by ksom** [comment]
@kiuka If you want to fix it you can change the version of @api-platform/admin to `^2.6.0` in your package.json and then run `docker-compose exec pwa yarn upgrade @api-platform/admin`.

**Comment by kiuka** [comment]
Thank you, the new version seems to be working well. The issue can be closed, I just wanna ask another question before I do it:

The client generator creates modules for the resources and according to the documentation (https://api-platform.com/docs/client-generator/) it should handle the validation errors and also should handle the relations but neither of it works.

I used the generator as it was in the documentation:
```
docker-compose exec pwa \
     generate-api-platform-client
```

which created the endpoints: /books, /reviews

When I try to create  a new book and send the empty form, I get this. It shows that the response was containing the validation errors, not sure if it should get it from there or no, but in the code it seems that it should show the errors per form element:

![image](https://user-images.githubusercontent.com/11283191/107224001-29064180-6a17-11eb-8fb7-d63e15deb4dc.png)

Also the review relation is a simple input instead of a selector, that what I would call handling relations:)

Am I missing something here or it isnt what I imagined?

Thanks!
